Fundamentals/Backoffice/Property-Editors/Built-in-Property-Editors/Block-Editor/label-property-configuration.md

```diff
@@ -0,0 +1,61 @@
+---
+---
```

Comment:
Which major versions of Umbraco does this article apply to?

Reply:
NC was brought in in 7.7, so I believe all apply from then